hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/ScheduledTaskDescriptor.java

```diff
@@ -38,11 +44,24 @@
     private ScheduledFuture<?> scheduledFuture;
 
     /**
-     * Stats are written from a single thread and read by many outside the partition threads. (see. Member owned tasks)
+     * Stats are written from a single thread and read by many outside the partition threads.
```

Comment:
I don't have a strong argument. The Atomic API might be clean but this was a surprise to me - I see the Atomic field and expect multiple writers only to realise later that that wasn't the case.